### DISCUSSION TITLE
Is the host adress safe?

### DISCUSSION THREAD

**Discussion by User13728** [community/community#121155] [Other Feature Feedback, Questions, & Ideas]
I would like to use SSH for authentication.
In my local repository I called `git remote add origin git@github.com:SSH_LINK` with no warnings or errors.
Then I called `git push --set-upstream origin BRANCH_NAME`, which returned:

> The authenticity of host 'github.com (140.82.XXX.X)' can't be established.
> ED25519 key fingerprint is SHA256:+DiY3wvvV6TuJJhbpZisF/zLDA0zPMSvHdkr4UvCOqU.
> This key is not known by any other names.
> Are you sure you want to continue connecting (yes/no/[fingerprint])?

How can I know if the IP adress above is safe before saying yes? Mozilla Firefox labels the IP address as "NOT secure".
I could not find the IP address in https://api.github.com/meta. But I can find the fingerprint in meta.

**Comment by niladri-1** [comment]
## To ensure the safety of the host address, follow these steps:

1. Verify the authenticity of the SSH key fingerprint provided in the warning message with the one listed in GitHub's meta.
2. Cross-reference with official GitHub resources or documentation to confirm the validity of the IP address.
3. Trust your instincts; if anything seems suspicious, refrain from proceeding until you're confident about the connection's authenticity.
